app/assets/javascripts/shop_orders.coffee

```diff
@@ -6,12 +6,12 @@ $ ->
       json_object = {}
       json_object["id"] = id
       json_object["status"] = $(this).find('select').val()
-      json_array.push(json_object) if id?
+      if id?
```

Comment:
@andela-anwocha You should write this on a line like this `if id then json_array.push(json_object)`